Makefile

```diff
@@ -66,11 +66,11 @@ SKIP_DOCKER_COMMAND ?= false
 
 ## -- Namespaces
 
-# namespace in which the global operator is deployed (see config/global-operator)
-GLOBAL_OPERATOR_NAMESPACE ?= elastic-system
-# namespace in which the namespace operator is deployed (see config/namespace-operator)
-NAMESPACE_OPERATOR_NAMESPACE ?= elastic-namespace-operators
-# comma separated list of namespaces in which the namespace operator should watch resources
+# namespace in which the operator is deployed (see config/global-operator)
```

Comment:
nit: not from your PR but I'm not sure where `config/global-operator` comes from

Reply:
There are still some references to `global` to delete in this Makefile.

```sh
153: # Run the operator locally with ... and operator namespace for a GLOBAL operator
159:  --development --operator-roles=GLOBAL,namespace \
172:  --operator-roles=GLOBAL,namespace \
193: # Deploy both the GLOBAL and namespace operators against the current k8s cluster
```